### ISSUE TITLE
[EN][Item] Vishanka, Jaws of the Earth

### ISSUE THREAD

**Issue by Psychobird7** [Atlantiss/BugTracker#10038]
**Description**: 
Vishanka, Jaws of the Earth (Item IDs 78480 (LFR), 78359 (Normal), 78471 (Heroic)) currently has two issues. 
It procs too often, **and** uses an incorrect crit multiplier. 
This is going to be fairly long, I apologise in advance. Will add a TLDR. 

**Current behaviour**: 

**Firstly:** Vishanka procs too much. 
As tested on PTR. 
As Survival, procs were observed on average every ~6.5s. 52 procs over 360s. 
As Marksmanship, procs were observed on avg every 5.5s. 68 procs over 360s. 

In a test run as MM, performing 5987 Auto shots, and 1726 Wild quiver procs, 3012 Speaking of Rage procs were observed. 
This is a 39% proc rate. While this test run was not entirely accurate because it fails to account for certain parameters such as refreshes, the observed proc rate is significantly higher than what it should be. 

**Secondly:** Vishanka currently uses a crit multiplier of 2.0 (2.06 if using the meta). 

**Expected behaviour**: 

**TLDR**: 
-- Vishanka should have a 15% proc chance on ranged hit, and a 15s ICD. 
-- Speaking of Rage should use a crit multiplier of 1.5, not 2.0. 
**/TLDR**

**Firstly:** Vishanka should proc much less than it currently does. 
The proc rate is modelled to be 15% on ranged hit on every tool available, which it currently does not have on Atlantiss. 
However **it should also have a Cooldown.** 

Simcraft shows both every hunter spec dealing ~1.7% of their total damage with the weapon's proc, and procs occuring at 20s intervals on average. Note that this data was gathered using 500k iteration runs, so RNG is out of the equation. 

A simulation run with a MM hunter profile modified to only auto-shot showed 16 procs over 360s, while the same MM hunter using their normal abilities showed 21.4 procs over 360s. 
Other specs showed similar behaviour. 
Simcraft results for MM: https://imgur.com/fm4U5uw 
Results for SV: https://imgur.com/o8IBcK3
~~**This clearly points to an ICD, or some form of proc chance reduction, or restrictions on abilities that can proc it.**~~ **Speaking of Rage definitely has an ICD, see the Yo'sahj logs below**. 

**Retail logs show proc rates that align with the rates modelled on Simcraft.** 
Pick any of the following Ultraxion logs. Longer fight durations are helpful, however a sufficiently large sample of logs will have the same effect. 
http://www.worldoflogs.com/ranking/eu-us/dragon-soul/ultraxion/heroic-25/dps/hunter/beast-mastery/ 
http://www.worldoflogs.com/ranking/eu-us/dragon-soul/ultraxion/heroic-25/dps/hunter/marksmanship/ 
http://www.worldoflogs.com/ranking/eu-us/dragon-soul/ultraxion/heroic-25/dps/hunter/survival/ 

To confirm this hypothesis, look through logs of Yor'sahj. Pick any of the following logs, and look for Speaking of Rage applications. 
http://www.worldoflogs.com/ranking/eu-us/dragon-soul/yorsahj-the-unsleeping/heroic-25/dps/hunter/beast-mastery/ 
http://www.worldoflogs.com/ranking/eu-us/dragon-soul/yorsahj-the-unsleeping/heroic-25/dps/hunter/marksmanship/ 
http://www.worldoflogs.com/ranking/eu-us/dragon-soul/yorsahj-the-unsleeping/heroic-25/dps/hunter/survival/ 

After looking through around 200 logs, the shortest average time between procs observed was 19.1s, with the average being 20-22s. 

The best estimate of an ICD based on these simulations and logs is 15s. These give enough time for shots to land, and cause a proc within 2s of the ICD being over if "lucky".
The shortest time between procs observed in logs was 17.2s. 

Additional evidence of ICD; 
http://www.worldoflogs.com/reports/rt-jjuajgaebmoyoqoe/xe/?s=830&e=1131&x=spellID+%3D+109858 
See the intervals between procs. 
Specifically, look at the instances of damage dealt to Forgotten Ones. This is 25H, so there are 11+ targets being Multishotted at. If there was no ICD, there would be at least one instance of a double proc, which there isn't. This further confirms the ICD hypothesis.  

**Secondly:** Speaking of Rage should use a crit multiplier of 1.5 (1.545 if using the meta). 
http://www.worldoflogs.com/reports/rt-jjuajgaebmoyoqoe/xe/?s=830&e=1131&x=spellID+%3D+109858 
"[19:24:45.703] Haffwalker Speaking of Rage Forgotten One 952              -- hit
[19:24:45.983] Haffwalker Speaking of Rage Forgotten One 1471"            -- crit 
952*1.545=1470.84 

**Steps to reproduce the problem**:

1. Get a hunter, get Vishanka 
2. Hit a dummy 
3. See you will get a lot of procs. 

**Revision**: 5771 (5763 on PTR)

**Comment by RangerTalion** [comment]
Using Tricks of the Trade the tooltip will update to 10690 damage
![wowscrnshot_051318_003343](https://user-images.githubusercontent.com/26337767/39961734-5fb0ac5e-5645-11e8-9773-78d3ce037fdb.jpg)

**Comment by Psychobird7** [comment]
A 15s ICD is plausible, yes. I will edit the original report to reflect this as I found instances of it proccing in 17s. 

Damage multipliers on the player seem to be meant to affect it, based on both logs, and Simcraft. 

One thing though. Never use a private server bugtracker as "evidence". Please.

**Comment by RangerTalion** [comment]
I'm trying to get reliable data aswell since I want it to work as intended, various sources such as comments in elitistjerks / wowhead back from 2012 or from other private servers indicate it has no ICD while other sources say that it does.
Right now testing it on both PTR + Live I get values of ~3-4 % of my DPS

On a 4,5 minute Shannox HC from yesterday I got these values
![wowscrnshot_051318_014223](https://user-images.githubusercontent.com/26337767/39962151-7606fa44-564f-11e8-9076-0b5c7de350c2.jpg)
![wowscrnshot_051318_014350](https://user-images.githubusercontent.com/26337767/39962152-76271c34-564f-11e8-85a1-dc1e96a06290.jpg)

**Comment by Psychobird7** [comment]
Additional evidence of ICD; 
http://www.worldoflogs.com/reports/rt-jjuajgaebmoyoqoe/xe/?s=830&e=1131&x=spellID+%3D+109858 
See the intervals between procs. 
Specifically, look at the instances of damage dealt to Forgotten Ones. This is 25H, so there are 11+ targets being Multishotted at. If there was no ICD, there would be at least one instance of a double proc, which there isn't. This further confirms the ICD hypothesis.

**Comment by Piklo** [comment]
should be fixed in [rev5782](https://atlantiss.eu/changelog#rev5782)
